### DISCUSSION TITLE
@defer support

### DISCUSSION THREAD

**Discussion by jpvajda** [the-guild-org/apollo-angular#1808] [Ideas]
👋

At Apollo we are currently working on implementing `@defer`support in our Web Client I wanted to check in to see if there was any questions we can answer to help this library leverage the use of `@defer` once we release it.

https://github.com/apollographql/apollo-client/issues/8184

**Comment by jpvajda** [comment]
👋  Just a follow up to my comment above: `@defer` is now supported in Apollo Web Client. You can see our[ docs ](https://www.apollographql.com/docs/react/data/defer)for more information.

**Comment by DavidSanwald** [comment]
Is there any news on this? On my project, we'd benefit greatly from @defer support.
Are there any current plans to implement @defer support for apollo-angular?
